fibrechannel/fibrechannel.go

```diff
@@ -1 +1,300 @@
 package fibrechannel
+
+import (
+	"fmt"
+	"github.com/j-griffith/csi-connectors/logger"
+	"io/ioutil"
+	"k8s.io/kubernetes/pkg/util/mount"
```

Comment:
My intent was to leave mount/format out of the connector and focus only on "connecting/attaching" the device to the node.  Maybe that's not the right way to go, but it seemed that it would be good to collapse mount/fmt into a single lib becuause it's shared across all the various connectors.  I also wanted to avoid sucking in a bunch of K8s code if possible.

Regardless, if we decide to put mount in here and we want to use k8s imports we're going to need to run dep init.  I had hoped to keep this all to just use std packages and avoid that if possible.